automatic/cmake/update.ps1

```diff
@@ -31,13 +31,27 @@ function global:au_GetLatest {
     $url32_portable = $allZips | Where-Object { $_ -match "$version-$re32" }
     $url64_portable = $allZips | Where-Object { $_ -match "$version-$re64" }
 
-    $streams.Add($versionTwopart, @{
-        Version = $version
-        URL32_i = [uri]$_
-        URL64_i = [uri]$url64
-        URL32_p = [uri]$url32_portable
-        URL64_p = [uri]$url64_portable
-      })
+    if ($streams.ContainsKey($versionTwoPart)) {
+        $previousKeyVersion = Get-Version $streams[$versionTwoPart].Version
```

Comment:
Not requiring a change, as it is fine.

But to note, you can store a version acquired form `Get-Version` from the hashtable (this will work with AU). If that is done you do not need to transform the version again every time.